rebar.config

```diff
@@ -68,7 +68,7 @@
         {aebytecode, {git, "https://github.com/aeternity/aebytecode.git", {ref, "10cc127"}}},
 
         {aeserialization, {git, "https://github.com/aeternity/aeserialization.git",
-                          {ref,"4a07297"}}},
+                          {ref,"d87adfd"}}},
```

Comment:
Note: Requires https://github.com/aeternity/aeserialization/pull/9 to be merged first.